src/text.rs

```diff
@@ -901,7 +901,7 @@ impl GlyphAtlas {
 
         // render glyph to image
         canvas.save();
-        canvas.reset();
+        canvas.hard_reset();
```

Comment:
This makes me wonder if perhaps `set_render_target` should be changed. I mean, won't pretty much **any** user of `set_render_target` when rendering into an image want to call `hard_reset()` to reset the dpi otherwise?